acquia/acquia_cms
# ACMS-788: Resolve issue to install & enable acquia_cms_page module independently.

Resolved issue to install & enable acquia_cms_page module without need to install acquia_site_studio module.
The error was coming due to config: `field.storage.node.field_layout_canvas`. When we enable acquia_cms_page module, it tries to import this optional config, even though we have added `cohesion_elements` module as a dependency for this config.

When further debugged, I found that, when drupal tries to calculate missing dependencies for this config and it doesn't get `cohesion_elements` as a missing dependency, so drupal assume that, there's no missing dependency for this config and it try to import this config. So, during import it report the following error:
```Unable to determine class for field type 'cohesion_entity_reference_revisions' found in the 'field.storage.0' configuration ```

**Proposed changes**
Looked for the issue in drupal 9 core issue tracker and found that this issue was reported ([https://www.drupal.org/node/3180588](https://www.drupal.org/node/3180588)) and a patch is added for the same to resolve this.


**Testing steps**
Install the acms site and you should not see any error and the module acquia_cms_page should be enabled.

**Merge requirements**
